src/orquestra/integrations/qiskit/backend/backend.py

```diff
@@ -315,6 +316,8 @@ def aggregate_measurements(
         """
         circuit_set_from_jobs = []  # circuits that qiskit ran
         circuit_set_from_batches = []  # circuits that users sent
+        self.list_virtual_to_physical_qubits_dict = []
```

Comment:
[question]: why is `list_virtual_to_physical_qubits_dict` a property to QiskitBackend if it's reset to an empty list during every call of this method? as in, it doesn't seem to be an inherent property of the backend (which adding it as a property seems to imply) but it's just a property of the current measurements. would it make more sense to return it from this method?

Reply:
You cannot return it from this method, as this would break the API.